modular_zubbers/code/modules/blood/blood.dm

```diff
@@ -0,0 +1,45 @@
+/obj/effect/decal/cleanable/blood
+	var/poolsize = 0
+
+/obj/effect/decal/cleanable/blood/Initialize(mapload)
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(blood_check)), 8 SECONDS, TIMER_UNIQUE)
```

Comment:
Just use START_PROCESSING if you plan on running something every x seconds like this